packages/core/test/acceptance/signal_debug_spec.ts

```diff
@@ -302,4 +313,42 @@ describe('getSignalGraph', () => {
       producer: fourFiveSixNode,
     });
   }));
+
+  it('should return no nodes or edges for a NodeInjector that only has directives and no component', fakeAsync(() => {
+    @Directive({
+      selector: '[myDirective]',
+      standalone: true,
+    })
```

Comment:
```suggestion
    @Directive({
      selector: '[myDirective]',
    })
```
Since we shipped v19, the explicit standalone: true isn't necessary anymore.